README.md

```diff
@@ -1 +1,29 @@
-# Hello-world
\ No newline at end of file
+# Hello-world 👏
```

Comment:
- [ ] Kindly make your README more descriptive and professional by adding the table of content to it as seen in the template, 
  - Add the Future Feature section to it and include 2-3 aspects of the project that you will add in the future
  - Kindly add the MIT license section to it and be sure the Link works perfectly by taking us to the licensed copyright you add in your project root directory.

Here is the new README template given to us by MIcroverse: [LINK](https://github.com/microverseinc/readme-template)

-N.B: Kindly remove the Microverse Logo at the top of the README file, optionally you can add your logo if you have one

Reply:
Hi @sanieni6, I'm your second reviewer: No You don't need a new pull request 👍